### PR TITLE
CLI for allele fraction plotting

### PR DESCRIPTION
**Pull request by davidbenjamin**

### CODE REVIEW

File: src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java [hunk 1]
```diff
@@ -53,8 +51,6 @@ public void testUnLoggedCommandLineSexChrs() throws IOException {
         final String sampleName = "HCC1143";
         File tDir = IOUtils.tempDir("Test", "Plotting");
         System.out.println(tDir);
```
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java]: There is never a reason to have a system.out.println.  Use logging, instead (examples are found in other CLIs)

File: src/test/java/org/broadinstitute/hellbender/utils/plotter/AlleleFractionSegmentedPlotterTest.java [hunk 2]
```diff
@@ -8,7 +8,7 @@
 import java.io.File;
 import java.io.IOException;
 
-public class AlleleFractionPlotterTest extends BaseTest {
+public class AlleleFractionSegmentedPlotterTest extends BaseTest {
     @Test
     public void testAlleleFractionPlotting() throws IOException {
```
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/utils/plotter/AlleleFractionSegmentedPlotterTest.java]: I do not like tests that throw IOExceptions.  I would argue that you should catch the IOException in the test and use:

```java
try {
    //.....snip....
} catch (final IOException ioe) {
    Assert.fail("Could not run test due to problems with a file", ioe);
}
```

File: src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java [hunk 3]
```diff
@@ -21,8 +21,6 @@ public void testUnLoggedCommandLine() throws IOException {
...
```
Comment (LeeTL1220) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java]: Can you use logging instead?

**Comment by LeeTL1220** [comment]
Minor stuff... back to you @davidbenjamin

File: src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java (hunk 1 above)
Comment (samuelklee) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java]: also note that i removed these in #330, so if you decide to log, then rebase accordingly.  although it looks to me like this was just debugging stuff that got left behind?

Comment (davidbenjamin) [inline src/test/java/org/broadinstitute/hellbender/tools/exome/PlotSegmentedCopyRatioIntegrationTest.java]: Debugging stuff that I wasn't sure whether to keep.  removed in this commit.

File: src/test/java/org/broadinstitute/hellbender/utils/plotter/AlleleFractionSegmentedPlotterTest.java (hunk 2 above)
Comment (davidbenjamin) [inline src/test/java/org/broadinstitute/hellbender/utils/plotter/AlleleFractionSegmentedPlotterTest.java]: Tried to do this and IntelliJ realized that `runCommandLine` never throws an `IOException`, so the `throw` was silly to begin with.  Removed -- and now the code looks like all the other integration tests in the repo which seem to throw only when there's an expected exception.

**Comment by davidbenjamin** [comment]
Back to @LeeTL1220

**Pull request merged**